tests/main/interfaces-bluez/task.yaml

```diff
@@ -11,7 +11,18 @@ execute: |
     if ! snap list --unicode=never bluez &> /dev/null ; then
         echo "Installing bluez snap from the store ..."
         expected="(?s)bluez .* from Canonical\\* installed\\n"
-        snap install bluez | grep -Pzq "$expected"
+        if snap install bluez > output 2> errors; then
+            grep -Pzq "$expected" output
```

Comment:
I'm also not sure what this is adding to the test, AFAICT it's not necessary but maybe someone else has some idea. I can see it's been there since the test was added but I didn't see any discussion about it.